stestr/commands/failing.py

```diff
@@ -103,7 +103,7 @@ def failing(repo_url=None, list_tests=False, subunit=False,
         for failures.
     :rtype: int
     """
-    repo = util.get_repo_open('file', repo_url)
+    repo = util.get_repo_open(repo_url)
```

Comment:
I'm curious does this work as expected? Aren't we passing `repo_url` here as the value for the `repo_type` argument. I think we're probably not catching this in tests because we use the default `None` and nothing is triggering the deprecation warning. I think this should be:

```suggestion
    repo = util.get_repo_open(repo_url=repo_url)
```

Reply:
Thanks, yeah, my bad.